com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -108,7 +108,9 @@ public Matrix4x4 GetGPUProjectionMatrix(int viewIndex = 0)
         public bool isHdrEnabled;
         public bool requiresDepthTexture;
         public bool requiresOpaqueTexture;
-
+#if ENABLE_VR && ENABLE_XR_MODULE
+        public bool xrRendering;
```

Comment:
we recently deprecated `isStereoEnabled`, shouldn't we use that instead?

Reply:
This is done by purpose. We want to deprecate isStereoEnabled and switch users to use xr.enabled instead. xr.enabled is populated by XR system every frame. 
We believe deprecation can make it clear to users that they should use xr.enabled for runtime XR check. As for xrRendering, this is populated by camera inspector UI, not runtime XRSystem.